drivers/docker/config.go

```diff
@@ -310,6 +310,8 @@ var (
 		// task containers.  If true, nomad doesn't start docker_logger/logmon processes
 		"disable_log_collection": hclspec.NewAttr("disable_log_collection", "bool", false),
 
+		"new_networking": hclspec.NewAttr("new_networking", "bool", false),
```

Comment:
You read my mind… Also I am not sure if the default (if omitted) wouldn't be always false and then it would be `no pause container = false` which would be the default we don't want? Or can we express that better in hcl (bool reference to allow checking for null?).

Reply:
Renamed to `nomad_native_networking` for the lack of a better name, feel free to change at will.